neuro_san/session/direct_concierge_session.py

```diff
@@ -56,7 +60,31 @@ def list(self, request_dict: Dict[str, Any]) -> Dict[str, Any]:
                 "agents" - the sequence of dictionaries describing available agents
         """
         agents_names: List[str] = self.network_storage.get_agent_names()
+        empty_list: List[str] = []
+
         agents_list: List[Dict[str, Any]] = []
         for agent_name in agents_names:
-            agents_list.append({"agent_name": agent_name, "description": ""})
-        return {"agents": agents_list}
```

Comment:
Above: Before we just listed names.
Below: Now we get metadata information from each agent network.